lib/gen/api/enums/v1/command_type_pb.rb

```diff
@@ -0,0 +1,35 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
How is the folder structure expected to line up with the module structure in Ruby? If I have `Temporal.Foo.Bar` where is that expected to be? Is it worth moving generated files to conform to that too?